web/src/main/resources/static/oncotree-mapping.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
+    <link type="text/css" rel="stylesheet" href="css/bootstrap.css"/>
+    <link type="text/css" rel="stylesheet" href="css/style.css"/>
+    <script type="text/javascript" src="js/jquery-1.11.1.min.js"></script>
+    <script type="text/javascript" src="js/bootstrap.js"></script>
+
+  </head>
+
+  <body>
+ 
+    
+
+<div class="container">

```

Comment:
style: &lt;body&gt; is indented .. shouldn't this &lt;div&gt; element be further indented? There is inconsistent indenting throughout this html file.